specification/ai/Azure.AI.Projects/agents/tools/tool_resources.tsp

```diff
@@ -169,9 +169,11 @@ model CreateFileSearchToolResourceVectorStoreOptions {
   @encodedName("application/json", "chunking_strategy")
   chunkingStrategy: VectorStoreChunkingStrategyRequest;
 
-  /** Set of 16 key-value pairs that can be attached to a vector store. This can be useful for storing additional information
-   * about the vector store in a structured format. Keys can be a maximum of 64 characters long and values can be a maximum of
-   *  512 characters long. */
+  /** 
+   * Set of 16 key-value pairs that can be attached to a vector store. This can be useful for storing additional 
```

Comment:
Should it say "up to 16 key-value pairs"? I'm assuming this is optional and can be any number up to and including 16

Reply:
updated